test/new-e2e/system-probe/connector/main.go

```diff
@@ -54,6 +56,7 @@ func readArgs() *args {
 	serverAliveCountPtr := flag.Int("server-alive-count", 560, "Maximum keep alive messages to send before disconnecting upon no reply")
 	sshFilePathPtr := flag.String("ssh-file", "", "Path to private ssh key")
 	vmCmd := flag.String("vm-cmd", "", "command to run on VM")
+	sendEnvVars := flag.String("send-env-vars", "", "Comma-separated list of environment variables to send through the connection")
```

Comment:
I don't think we need this, because we already have the configure the allowed variables on the server side. Can we change it in test-infra-definitions instead?